flink-runtime/src/test/java/org/apache/flink/runtime/metrics/util/MetricUtilsTest.java

```diff
@@ -73,7 +70,7 @@ public void cleanupReferencedObjects() {
      * RpcSystem)} respects the given {@link MetricOptions#QUERY_SERVICE_THREAD_PRIORITY}.
```

Comment:
Would you mind fix this broken JavaDoc into a hotfix commit?

Reply:
fixed